.gitignore

```diff
@@ -0,0 +1,8 @@
+/.idea*
+/build*
+/install*
+/sandbox*
+/llvm*
+/mlir_aie*
+.vscode
+__pycache__
```

Comment:
Missing EOL in a few files.
You do not like my https://github.com/Xilinx/mlir-aie/blob/main/.editorconfig ? :-)

Reply:
If you flip to this branch you'll see there's nothing at the root - except for `.vscode` (a mistake) these are the only generated artifacts.